play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukSelect.scala.html

```diff
@@ -43,7 +43,7 @@
       @{formGroup.beforeInput.map(_.asHtml)}
       <select class="@toClasses("govuk-select", classes, errorMessage.fold("")(_ => "govuk-select--error"))" @attrs>
         @for(item <- items) {
-          @defining((value.nonEmpty && item.value == value) || item.selected) { isSelected =>
+          @defining((value.nonEmpty && value.exists(_.nonEmpty) && item.value == value) || item.selected) { isSelected =>
```

Comment:
I think we can drop the initial value.nonEmpty and just have the exists(_.nonEmpty) I think if you .exists on an Option that is None then it will return false too